client/my-sites/customer-home/index.js

```diff
@@ -0,0 +1,18 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import page from 'page';
+
+/**
+ * Internal dependencies
+ */
+import { navigation, siteSelection, sites } from 'my-sites/controller';
+import home from './controller';
+import { makeLayout, render as clientRender } from 'controller';
+
+export default function() {
+	page( '/home', siteSelection, sites, makeLayout, clientRender );
+
+	page( '/home/:site_id', siteSelection, navigation, home, makeLayout, clientRender );
```

Comment:
I know this is primarily focused on layout at the moment, but should we add a redirect here if `canCurrentUserUseCustomerHome` is false?

Reply:
As per our discussion, will add a message in the section itself, informing the user about this.